tests/Resources/ti.ui.test.js

```diff
@@ -0,0 +1,239 @@
+			should(Ti.UI.convertUnits('1mm', Ti.UI.UNIT_IN)).be.approximately(0.0393701, 0.0001);
+		});
+
+		// mm -> px
+		// since 25.4mm/2.54cm = 1in, then in pixels it should equal dpi (dots per inch!)
+		it('returns 25.4 mm = dpi pixels', function () {
+			var dpi = Ti.Platform.displayCaps.dpi;
+			should(Ti.UI.convertUnits('25.4mm', Ti.UI.UNIT_PX)).be.approximately(dpi, 0.0001);
+		});
+
+		// px -> cm
+		// since 2.54cm = 1in, dpi pixels = 1in, so dpi px should = 2.54cm
+		it('returns dpi pixels = 2.54 cm', function () {
+			var dpi = Ti.Platform.displayCaps.dpi;
+			should(Ti.UI.convertUnits(dpi + 'px', Ti.UI.UNIT_CM)).be.approximately(2.54, 0.0001);
+		});
+
+		// px -> dip
+		it('returns Ti.Platform.DisplayCaps.logicalDensityFactory pixels = 1 dip', function () {
+			var logicalDensityFactor = Ti.Platform.displayCaps.logicalDensityFactor;
+			should(Ti.UI.convertUnits(logicalDensityFactor + 'px', Ti.UI.UNIT_DIP)).be.approximately(1, 0.001);
+		});
+
+		// px -> in
+		// If we try to convert dpi pixels to inches, it should be 1 inch (because dpi itself is pixels per inch)
+		it('converts dpi pixels = 1 in', function () {
+			var dpi = Ti.Platform.displayCaps.dpi;
+			should(Ti.UI.convertUnits(dpi + 'px', Ti.UI.UNIT_IN)).eql(1);
+		});
+
+		// px -> mm
+		// since 2.54cm = 1in, dpi pixels = 1in, so dpi px should = 25.4cm
+		it('returns dpi pixels = 25.4 mm', function () {
+			var dpi = Ti.Platform.displayCaps.dpi;
+			should(Ti.UI.convertUnits(dpi + 'px', Ti.UI.UNIT_MM)).be.approximately(25.4, 0.0001);
+		});
+
+		// TODO try "converting" a unit to itself, assume we get exact same value!
+
+		it.androidBroken('returns 0 if percentage used', function () {
+			should(Ti.UI.convertUnits('100%', Ti.UI.UNIT_CM)).eql(0); // FIXME Android "expected -0.015875 to equal 0"
+			should(Ti.UI.convertUnits('5%', Ti.UI.UNIT_DIP)).eql(0);
+			should(Ti.UI.convertUnits('12%', Ti.UI.UNIT_IN)).eql(0);
+			should(Ti.UI.convertUnits('65%', Ti.UI.UNIT_MM)).eql(0);
+			should(Ti.UI.convertUnits('20%', Ti.UI.UNIT_PX)).eql(0);
+		});
+
+		// 1097
+		it('returns 0 if non-numeric value passed in', function () {
+			should(Ti.UI.convertUnits('abc', Ti.UI.UNIT_CM)).eql(0);
+			should(Ti.UI.convertUnits('abc', Ti.UI.UNIT_DIP)).eql(0);
+			should(Ti.UI.convertUnits('abc', Ti.UI.UNIT_IN)).eql(0);
+			should(Ti.UI.convertUnits('abc', Ti.UI.UNIT_MM)).eql(0);
+			should(Ti.UI.convertUnits('abc', Ti.UI.UNIT_PX)).eql(0);
+		});
+
+		// TODO Test that 'dp' and 'dip' suffix are both UNIT_DIP
+	});
+
+	// Constants are tested in ti.ui.constants.test.js
+
+	// TODO Write tests for Ti.UI.global properties below!
+	it('backgroundColor');
+	it('backgroundImage');
+	it('currentTab');
+
+	// FIXME Get working on iOS
+	// FIXME Get working on Android - Ti.UI.currentWindow is null
+	// Supposedly this property should only exist when using Ti.UI.Window.url to load JS files into own context! But we support elsewhere for Windows
+	it.androidAndIosBroken('.currentWindow', function (finish) {
```

Comment:
This property only existed before we removed the `Ti.UI.Window.url` property and was forgotten during the docs update, see [here](https://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI-property-currentWindow) for reference.